cf/i18n/resources/en_US.all.json

```diff
@@ -910,13 +910,28 @@
       "modified": false
    },
    {
+      "id": "Restart a single app:\n",
+      "translation": "Restart a single app:\n",
+      "modified": false
+   },
+   {
       "id": "CF_NAME restage APP_NAME",
       "translation": "CF_NAME restage APP_NAME",
       "modified": false
    },
    {
-      "id": "CF_NAME restart APP_NAME",
-      "translation": "CF_NAME restart APP_NAME",
+      "id": "   CF_NAME restart APP_NAME\n",
```

Comment:
Have those strings changed? Why are we not seeing the same changes for the other language files?

I am assuming they were not changed but somehow the `en_US` file was touched? Otherwise, the other files need to change as well.